core/src/test/resources/packaging/scripts/25_tar_plugins.bats

```diff
@@ -65,7 +65,7 @@ setup() {
     [ -e "$MARVEL_ZIP" ]
 
     # Install Marvel
-    run /tmp/elasticsearch/bin/plugin -i elasticsearch/marvel/latest -u "file://$MARVEL_ZIP"
+    run /tmp/elasticsearch/bin/plugin install elasticsearch/marvel/latest -u "file://$MARVEL_ZIP"
```

Comment:
Can you please also remove the 2 tests in this file that install Marvel?

Nothing to do with your changes but Marvel 1 does not work on 2.0.